lib/attributes/backend/dpcpp/shared.cpp

```diff
@@ -11,14 +11,55 @@ namespace {
 using namespace oklt;
 using namespace clang;
 
-// TODO: There is no TypeDecl handler in DPCPP backend
-HandleResult handleSharedAttribute(SessionStage& s, const VarDecl& var, const Attr& a) {
+std::string getCleanTypeString(SessionStage& s, QualType t) {
```

Comment:
Better to move the function in some utils file.
And looks like SessionStage is unused.